README.md

```diff
@@ -0,0 +1,118 @@
+Aubrey-transcription [![Build Status](https://www.travis-ci.com/unt-libraries/aubrey-transcription.svg?branch=master)](https://www.travis-ci.com/unt-libraries/aubrey-transcription)
+====================
+
+
+About
+--------------------
+
+This is a service that returns information about available transcription files
+for a given record. The data is returned as JSON. This service is intended to
+be placed on the same server that the actual transcription files are located on,
+and it must have access to those files, which must be in a pairtree directory
+structure.
+
+
+Requirements
+--------------------
+
+* Python 2.7, 3.4 - 3.6
+* Flask 1.0.2
+* pypairtree 1.1.0
+
+
+Developing/Testing
+--------------------
+
+In order to start developing or run the tests, it is recommended that you set up
+a virtual environment specifically for aubrey-transcription. To do so, download
+virtualenv and then set up a virtual environment with Python 2.7 according to the
```

Comment:
Should we remove the specification of 2.7 here, since you have now broadened compatibility?

Reply:
yes, yes we should